crates/sovereign-sdk/rollup-interface/src/node/rpc/mod.rs

```diff
@@ -207,6 +207,8 @@ pub struct SoftBatchResponse {
     /// Public key of the signer
     #[serde(with = "hex::serde")]
     pub pub_key: Vec<u8>,
+    /// Deposit data from the L1 chain
+    pub deposit_data: Vec<Vec<u8>>,
```

Comment:
checkout how txs is serialized here. serialize deposit_data the same way

Reply:
Txs are not directly Vec<u8>, so you need some tweak here. Will do it later.